utils/msl/memberships.py

```diff
@@ -0,0 +1,127 @@
+"""Module for checking membership status."""
+
+import contextlib
+import logging
+from typing import TYPE_CHECKING
+
+import aiohttp
+import bs4
+from bs4 import BeautifulSoup
+
+from config import settings
+from exceptions import MSLMembershipError
+from utils import GLOBAL_SSL_CONTEXT
+
+if TYPE_CHECKING:
+    from collections.abc import Mapping, Sequence
+    from logging import Logger
+    from typing import Final
+
+
+__all__: "Sequence[str]" = (
+    "fetch_community_group_members_count",
+    "fetch_community_group_members_list",
+    "is_id_a_community_group_member",
+)
+
+
+BASE_SU_PLATFORM_WEB_HEADERS: "Final[Mapping[str, str]]" = {
+    "Cache-Control": "no-cache",
+    "Pragma": "no-cache",
+    "Expires": "0",
+}
+
+
+BASE_SU_PLATFORM_WEB_COOKIES: "Final[Mapping[str, str]]" = {
+    ".ASPXAUTH": settings["SU_PLATFORM_ACCESS_COOKIE"],
+}
+
+
+MEMBERS_LIST_URL: "Final[str]" = f"https://guildofstudents.com/organisation/memberlist/{settings['ORGANISATION_ID']}/?sort=groups"
+
+
+_membership_list_cache: set[int] = set()
+
+
+logger: "Final[Logger]" = logging.getLogger("TeX-Bot")
```

Comment:
```suggestion
logger: "Final[Logger]" = logging.getLogger("TeX-Bot")


BASE_SU_PLATFORM_WEB_HEADERS: "Final[Mapping[str, str]]" = {
    "Cache-Control": "no-cache",
    "Pragma": "no-cache",
    "Expires": "0",
}

BASE_SU_PLATFORM_WEB_COOKIES: "Final[Mapping[str, str]]" = {
    ".ASPXAUTH": settings["SU_PLATFORM_ACCESS_COOKIE"],
}

MEMBERS_LIST_URL: "Final[str]" = f"https://guildofstudents.com/organisation/memberlist/{settings['ORGANISATION_ID']}/?sort=groups"


_membership_list_cache: set[int] = set()
```
Whitespace can be cleaned up a bit here to group related variables.